### PR TITLE
ConnectorMeshComponent

### PR DESCRIPTION
**Pull request by makimenko**
- AbstractLazyObject3D for Loaders and Text font loaders and etc
- ConnectorMeshComponent connecting line from source to object + watch position adjustments
- Added dependency: [three.meshline](https://github.com/spite/THREE.MeshLine)  **to be checked if ok**

image

### CODE REVIEW

**Comment by ComFreek** [comment]
The Travis bulid failed due to a missing semicolon in the `ng lint` step. Let's see if it works now.

File: projects/atft/src/lib/objects/text/text-mesh.component.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          height: this.height,
+          curveSegments: this.curveSegments,
+          bevelEnabled: this.bevelEnabled,
+          bevelThickness: this.bevelThickness,
+          bevelSize: this.bevelSize,
+          bevelOffset: this.bevelOffset,
+          bevelSegments: this.bevelOffset
+        });
+        const material: THREE.MeshBasicMaterial = this.getMaterial();
+        setTimeout(function () {
```
Comment (makimenko) [inline projects/atft/src/lib/objects/text/text-mesh.component.ts]: remove

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "core-js": "^3.2.1",
     "rxjs": "^6.5.2",
     "three": "^0.108.0",
+    "three.meshline": "^1.2.0",
```
Comment (makimenko) [inline package.json]: @ComFreek what do you think about adding [three.meshline](https://github.com/spite/THREE.MeshLine) dependency? Or it's better to keep "clean" repo without additional libraries?

Comment (ComFreek) [reply package.json]: @makimenko I think users of atft don't ship mission-critical software anyway. I think #1 use case for atft: prototyping and playing around, perhaps creating some cool tech demo (do you agree?). Hence I suppose a thresh.meshline dependency would be OK.

File: projects/atft/src/lib/objects/text/text-mesh.component.ts (hunk 1 above)
Comment (ComFreek) [reply projects/atft/src/lib/objects/text/text-mesh.component.ts]: @makimenko I am afraid I don't understand this comment.

Comment (makimenko) [reply projects/atft/src/lib/objects/text/text-mesh.component.ts]: sorry for confusion. That's for me :-)
I must remove this temporary sleep period before merge to master)

**Pull request merged**